internal/serialization/compact_serializer.go

```diff
@@ -88,11 +86,27 @@ func (c CompactStreamSerializer) IsRegisteredAsCompact(t reflect.Type) bool {
 	return ok
 }
 
-func (c CompactStreamSerializer) getOrReadSchema(input pubserialization.DataInput) Schema {
+func (c CompactStreamSerializer) getOrReadSchema(ctx context.Context, input pubserialization.DataInput) *Schema {
 	schemaId := input.ReadInt64()
-	schema, ok := c.ss.Get(schemaId)
+	schema, ok := c.ss.Get(ctx, schemaId)
 	if !ok {
 		panic(hzerrors.NewSerializationError(fmt.Sprintf("the schema cannot be found with id: %d", schemaId), nil))
 	}
 	return schema
 }
+
+func MakeSchemasFromConfig(cfg pubserialization.CompactConfig) map[int64]*Schema {
+	r := map[int64]*Schema{}
+	for _, ser := range cfg.Serializers() {
+		s := makeSchemaFromSerializer(ser)
+		r[s.ID()] = s
+	}
+	return r
+}
+
+func makeSchemaFromSerializer(ser pubserialization.CompactSerializer) *Schema {
+	sw := NewSchemaWriter(ser.TypeName())
+	// create the zero value for the type in the serializer and
```

Comment:
minor: you might forget to finish this sentence

Reply:
Completed at: https://github.com/hazelcast/hazelcast-go-client/pull/910/commits/024d0660e11937c621ffe9c764c9fd6e002d9231